README.md

```diff
@@ -62,6 +67,14 @@ The following properties are used if your app has an API:
 
 This is the list of API versions your app can use. Since `v1` is the default, you'll usually want at least that one defined.
 
+#### app_id.api.subItems
+
+If your API consists of multiple APIs you can list them in here it has the same signature as `{app_id.api}`.
+
+#### app_id.api.apiName
+
+If your API is accessible on oher URL than `/api/{app_id}/{versions[0]}/openapi.json` you can change this by passing correct name, the URL will look like `/api/{apiName}/{versions[0]}/openapi.json`
```

Comment:
Grammatical change:
"If your API is accessible on a URL other than `/api/{app_id}/{versions[0]}/openapi.json`, you can change it by passing the correct name. The URL will look like `/api/{apiName}/{versions[0]}/openapi.json`."

Reply:
Thank you! Should be fixed.